package.json

```diff
@@ -64,7 +64,7 @@
     "app-root-path": "^2.1.0",
     "asciify": "^1.3.5",
     "async-exit-hook": "^2.0.1",
-    "axios": "0.19.0-beta.1",
+    "axios": "0.18.0",
```

Comment:
I've changed this to `0.18.0` because `axios-mock-adapter` doesn't resolve with `0.19.0-beta-1` version specified making the test execution fail when run locally on my machine.

Reply:
I think we require the 0.19.0-beta.1 to fix various proxying issues. Although the failures you are seeing may be due to proxy issues.